public/api/conf/1.0/application.raml

```diff
@@ -41,7 +41,7 @@ uses:
             type: string
             required: false
             example: ABC123
-        X-EORI-Identfier:
+        X-Submitter-Identfier:
             description: Required for CSP users. Must be alphanumeric string with a maximum length of 12.
```

Comment:
why is the max length here stated as `12` when the validation regex has max length of `17`?

Reply:
fixed, should be 17